gui/src/app/components/series-list/_series-list.scss

```
@@ -0,0 +1,24 @@
+@import '../../assets/css/variables';
+
+.series-list {
+  &__item {
+    display: inline-block;
+    border: 1px solid $color-mid-dark;
+    border-radius: 4px;
+    font-size: 0.875rem;
+    font-weight: 500;
+    color: $color-mid-dark;
+    padding: 0 0.5rem;
```

This fixes the height issue for @spencerbygraves 
```suggestion
    padding: 0.1rem 0.5rem;
```